commands/interpolate_test.go

```diff
@@ -0,0 +1,222 @@
+package commands_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"runtime"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+	. "github.com/onsi/gomega/ghttp"
+)
+
+var (
+	session      *gexec.Session
+	templateFile *os.File
+	templateText string
+	err          error
+)
+
+var _ = Describe("interpolate", func() {
+	BeforeEach(func() {
+		login()
+		templateFile, err = ioutil.TempFile("", "credhub_test_interpolate_template_")
+	})
+
+	Context("shared command examples", func() {
```

Comment:
I'm not exactly sure what `shared command examples` means, is there other phrasing we could use?

Reply:
We noted a use of a [Ginkgo Shared Examples Pattern](http://onsi.github.io/ginkgo/#shared-example-patterns) and wanted to scope some of our test setup explicitly for use with the shared example that required it. (We would have put said setup in a BeforeEach as is proper,  We probably could have used `Describe` instead of `Context`. If we change this to `Describe("behavior shared with all other commands",`, would that be clear and satisfactory?